.hound.yml

```diff
@@ -0,0 +1,2 @@
+eslint:
+  version: 5.7.0
```

Comment:
add an end of line after end of file please!

Reply:
Thanks for pointing that out @andela-amutava